caliper/utils/command.py

```diff
@@ -115,6 +115,23 @@ def wget_and_extract_zip(url, download_to, chunk_size=1024):
 
     return download_to, download_root, download_dir
 
+def wget_and_extract_conda(url, download_to, chunk_size=1024):
+    """
+    Get an extract a .conda archive.
+    """
+
+    import conda_package_handling.api as cph
```

Comment:
Good call on placing this internal to the function (I would have suggested that so it's only imported when needed)!